.github/workflows/deploy-preview.yml

```
@@ -0,0 +1,108 @@
+name: Deploy Preview to Staging
+
+on:
+  push:
+    branches:
+      - 'feature/**'
+      - 'fix/**'
+
+jobs:
+  test:
+    name: Run Tests
+    uses: ./.github/workflows/test.yml
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/workflows/test.yml": open /home/jailuser/git/.github/workflows/test.yml: no such file or directory

(workflow-call)